invenio_theme/assets/semantic-ui/less/invenio_theme/theme/views/item.overrides

```diff
@@ -19,6 +19,16 @@
 }
 
 .ui.items > .item {
+    &.community-header {
```

Comment:
the custom CSS is not needed, I would prefer to wrap the description in a grid column